io/genbank2/genbank.go

```diff
@@ -0,0 +1,396 @@
+package genbank
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/TimothyStiles/poly/io/poly"
+	"io"
+	"regexp"
+	"strings"
+)
+
+func Parse(r io.Reader) (poly.Sequence, error) {
+	scanner := bufio.NewScanner(r)
+
+	// Create meta struct
+	var meta = poly.Meta{}
```

Comment:
It'll be good have references for the file format and potentially any reference parsers